src/ts/greedy-nav/index.ts

```diff
@@ -1,7 +1,12 @@
+// this is just a placeholder file
+
 import { Config, defaultConfig } from './Config';
 
 class GreedyNav {
     constructor(config: Config) {
         Object.assign(defaultConfig, config);
+        console.log('greedy-nav loaded');
```

Comment:
Can we remove this?

Reply:
Sure - it was just a non-invasive way of showing this works - but np. I'll remove it.